java-client/weather-server-example/build.gradle

```diff
@@ -0,0 +1,37 @@
+plugins {
+    id 'java'
+    id 'application'
+}
+
+dependencies {
+    implementation project(':java-client-flight-dagger'),
+            project(':java-client-barrage'),
+            project(':Util'),
+            project(':engine-table'),
+            project(':extensions-barrage'),
+            'com.intellij:forms_rt:7.0.3'
+
+    implementation(project(':grpc-java')) {
```

Comment:
you should not need to depend on our grpc-java - it doesn't provide any classes at all